lib/src/widgets/shape.dart

```diff
@@ -78,12 +118,17 @@ class _ArrowPainter extends CustomPainter {
     final arrowOffset =
         Offset(arrowHeight * math.cos(angle), arrowHeight * math.sin(angle));
 
+    final paint = Paint()
```

Comment:
I'd prefer to keep the arrow painter as it was, such a change is not part of this PR